README.md

```diff
@@ -47,15 +46,18 @@ git clone git://github.com/kartoza/docker-postgis
 To create a running container do:
 
 ```
-docker run --name="backups"\
-           --hostname="pg-backups" \
-           --link=watchkeeper_db_1:db \
+docker run --name="backup" \
```

Comment:
maybe name = "sftp-backup" sill be more obvious as the difference between ``backup`` and ``backups`` is very nuanced...